### ISSUE TITLE
Withings will deprecate access and refresh tokens endpoints as of September 1, 2021

### ISSUE THREAD

**Issue by fnsign** [kodymallory/MMM-withings#10]
Hi,

I recently received an email from the Withings Community that announces the deprecation of the tokens endpoints as of September 1, 2021:
_As part of our continual improvement, we are migrating the OAuth 2.0 endpoints. The deprecated services will continue to operate normally until September 2021. Please review the article below to get more information on the breaking change._
[Link to Details](https://support.withings.com/hc/en-us/articles/360016745358-Deprecating-access-and-refresh-tokens-endpoints)

I suppose that this change will impact the module, too, and that it will need adjustments. Do you intend to adapt the module accordingly? Unfortunally I'm not code savvy enough to support.

**Comment by kodymallory** [comment]
Thanks for alerting me! I'll make the change per [this notification on the withings support page](https://support.withings.com/hc/en-us/articles/360016745358--BREAKING-Deprecating-access-and-refresh-tokens-endpoints)

**Comment by kodymallory** [comment]
I started looking into this and ran into a series of issues. I discovered that the current APP that the module is authenticating against is setup as developer and thus is limited to only like 15 users (which we are also at) and secondly, that changing it to a type that allows up to 1000 users would break the authentication mechanism as it will not allow for the localhost as the return address. 

I haven't been looking at it too much, but started dabbling in how to setup an webservice that could host the authentication. Haven't made it very far.

I think I can change over to the proper URI per the notification, but might have revert the directions to the more manual initial authentication.
